documentation/source/development/benchmarking.rst

```diff
@@ -0,0 +1,7 @@
+Benchmarking
+============
+
+- Inexor will use `Google Benchmark <https://github.com/google/benchmark>`__ in the future.
+- Google Benchmark is also `available in conan center <https://conan.io/center/benchmark>`__, just as `Google Test <https://github.com/google/googletest>`__.
+- Benchmarks can also not run in GitHub actions since testing Vulkan features would require a graphics card.
```

Comment:
Not true, only partially, we can run our own additional Github Action Instances which can have a GPU.

Reply:
Wow! That's amazing!